tensorflow/compiler/xla/tests/llvm_compiler_test.cc

```diff
@@ -138,7 +142,11 @@ class CpuCompilerTest : public LLVMCompilerTest {
 
 class GpuCompilerTest : public LLVMCompilerTest {
  public:
+#if GOOGLE_CUDA
```

Comment:
`#if !TENSORFLOW_USE_ROCM` . let's try avoid `GOOGLE_CUDA` within XLA, as there really is no CUDA within XLA.